benches/bench.rs

```diff
@@ -207,6 +207,21 @@ fn build_cloneable(b: &mut Bencher) {
     });
 }
 
+fn access_column(b: &mut Bencher) {
+    let mut world = World::new();
+    let _enta = world.spawn((Position(0.0), Velocity(0.0)));
+    let _entb = world.spawn((true, 12));
+    let entc = world.spawn((Position(3.0),));
+    let _entd = world.spawn((13, true, 4.0));
+    let column = world.column::<Position>();
+
+    b.iter(|| {
+        for i in 0..100 {
```

Comment:
Why loop here?

Reply:
> For applications which do large amounts of random access for the same component types
the issue stated this as a problem, so I wanted to test _large access_ 😸

Reply:
`iter` will already run the supplied closure many times to compute the average performance, so this isn't necessary.